invokeai/frontend/web/src/features/ui/layouts/shared.ts

```diff
@@ -20,10 +20,10 @@ export const LEFT_PANEL_MIN_SIZE_PX = 420;
 export const RIGHT_PANEL_MIN_SIZE_PX = 420;
 
 export const BOARD_PANEL_MIN_HEIGHT_PX = 36;
-export const BOARD_PANEL_DEFAULT_HEIGHT_PX = 232;
+export const BOARD_PANEL_DEFAULT_HEIGHT_PX = 300;
 
 export const GALLERY_PANEL_MIN_HEIGHT_PX = 36;
-export const GALLERY_PANEL_DEFAULT_HEIGHT_PX = 232;
+export const GALLERY_PANEL_DEFAULT_HEIGHT_PX = 200;
```

Comment:
This changes the default sizes for these two panels on _all_ tabs. Is that the intention?

If not, and only the canvas tab needs special handling, create new constants for it.